src/search/sr_optimizers.c

```
@@ -0,0 +1,333 @@
+  return 1;
+}
+
+void sr_optimizer_config_from_env(sr_optimizer_config *cfg)
+{
+  if (!cfg) return;
+  (void)sr_optimizer_read_int_env("CSEARCH_MAX_EVALS", &cfg->max_evals);
+  (void)sr_optimizer_read_int_env("CSEARCH_MAX_ITERS", &cfg->max_iters);
+  (void)sr_optimizer_read_seed_env("CSEARCH_SEED", &cfg->seed);
+}
+
+void sr_optimizer_config_apply(const sr_optimizer_config *cfg)
```

Clang (14.0.6)

[warning] parameter 'cfg' is unused

(misc-unused-parameters)